Annotation/polar_golden_set.json

```diff
@@ -804,13 +877,25 @@
             {
                 "predicate": [
                     {
-                        "str": "clear",
+                        "str": "'s not even clear",
                         "lemma": "",
                         "POS": ""
                     }
                 ],
                 "type": "polar",
-                "clause": "",
+                "clause": "whether, or how, the NCAA monitors clustering",

```

Comment:
This is another case where the clause is the antecedent of the subject, and perhaps therefore not an embedded clause at all

Reply:
I think we just disagree on this one so will leave these future questions and think we should ask our supervisors